src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -693,6 +705,16 @@ private BaseElementFacade geDeleteActivityIcon(String activity) {
                                           activity));
   }
 
+  private BaseElementFacade getPinActivityIcon(String activity) {
+    return findByXPathOrCSS(String.format("//div[contains(@class,'contentBox')]//*[contains(text(),'%s')]//preceding::*[@class='v-list-item__title pl-3' and contains(text(),'Pin')]",
+            activity));
+  }
+
+  private BaseElementFacade getUnpinActivityIcon(String activity) {
+    return findByXPathOrCSS(String.format("//div[contains(@class,'contentBox')]//*[contains(text(),'%s')]//preceding::*[@class='v-list-item__title pl-3' and contains(text(),'Unpin')]",
```

Comment:
IMO the xpath is not pertient , let's try this one .
//*[contains(@class,'contentBox')]//*[contains(text(),'%s')]//preceding::*[contains(@class,'mdi-pin-off')]